RedacteurPortaal.Tests/Grains/NewsItemGrainTests.cs

```diff
@@ -40,9 +40,9 @@ public async Task CanAddNewsItemCorrectly()
                                          DateTime.UtcNow,
                                          Category.STORY,
                                          Region.LOCAL,
-                                         Array.Empty<MediaVideoItem>(),
-                                         Array.Empty<MediaAudioItem>(),
-                                         Array.Empty<MediaPhotoItem>());
+                                         new List<MediaVideoItem>(),
```

Comment:
Hier ook minimaal 1 item toevoegen aan iedere list, dan zien we wat er met de data gebeurt